ndk/src/audio.rs

```diff
@@ -434,20 +483,17 @@ impl AudioResult {
     }
 }
 
+// TODO: Delete this and `impl Error for AudioResult`, just like `MediaError`!
```

Comment:
I've just tackled it immediately in https://github.com/rust-mobile/ndk/pull/467 instead, thanks for reminding me!

In hindsight it could have been done in this PR as we're already breaking the `AudioError` type anyway.